starlette/responses.py

```diff
@@ -229,7 +229,13 @@ def __init__(
         self.background = background
         self.init_headers(headers)
         if self.filename is not None:
-            content_disposition = 'attachment; filename="{}"'.format(self.filename)
+            content_disposition_filename = quote(self.filename)
```

Comment:
I'm afraid this expression `quote("ä") == "ä"` will return false and interpret the encoding as a `latin-1`.